presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -103,6 +103,7 @@ public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> ta
                     .put("hive.time-zone", TIME_ZONE.getID())
                     .put("hive.security", security)
                     .put("hive.max-partitions-per-scan", "1000")
+                    .put("hive.assume-canonical-partition-keys", "true")
```

Comment:
There was a test that tested non-canonical partition keys but it's been disabled for over a year.  But when that gets reenabled, we can set this to false for that test.